apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/application-layer.graphqls

```diff
@@ -0,0 +1,37 @@
+# ApplicationNode represents this node is under monitoring by agent.
+type ApplicationNode implements Node {
+    id: ID!
+    name: String!
+    type: String
+    # Success rate of all incoming requests.
+    # Max value is 100.
+    # 2 Digits after floating point.
+    sla: Float!
+    # The number of incoming calls
+    calls: Long!
+    # ref: http://www.apdex.org/
+    # Max value is 1
+    # 2 Digits after floating point.
+    apdex: Float!
+    # The number of servers in the application code
+    numOfServer: Int!
+    # The number of servers alerting
+    numOfServerAlarm: Int!
+    # The number of services alerting
+    numOfServiceAlarm: Int!
+}
+
+# The conjectural node generated by exit span
+type ConjecturalNode implements Node {
+    id: ID!
+    name: String!
+    type: String
+}
+
+
+extend type Query {
+  getAllApplication(duration: Duration!): [ApplicationNode]
+  getApplicationTopology(appId: ID!, duration: Duration!): Topology
+  getSlowService(appId: ID!, duration: Duration!): [DurationItem]
```

Comment:
The return object name (DurationItem) of this method is an abstractly name. I think SlowService is better suits than DurationItem.